go/test/endtoend/onlineddl/scheduler/onlineddl_scheduler_test.go

```diff
@@ -539,6 +540,8 @@ func testScheduler(t *testing.T) {
 	t.Run("ALTER both tables, elligible for concurrenct, with throttling", func(t *testing.T) {
 		onlineddl.ThrottleAllMigrations(t, &vtParams)
 		defer onlineddl.UnthrottleAllMigrations(t, &vtParams)
+		onlineddl.CheckThrottledApps(t, &vtParams, throttlerapp.OnlineDDLName, true)
```

Comment:
Adding a "wait-for", because `onlineddl.ThrottleAllMigrations` is now an async flow.